org/mozilla/jss/tests/TestCertificationRequest.java

```diff
@@ -0,0 +1,33 @@
+package org.mozilla.jss.tests;
+
+import java.io.*;
+import java.security.KeyPair;
+
+import org.mozilla.jss.*;
+import org.mozilla.jss.asn1.*;
+import org.mozilla.jss.crypto.*;
+import org.mozilla.jss.pkcs10.*;
+import org.mozilla.jss.pkix.primitive.*;
+
+public class TestCertificationRequest {
```

Comment:
Do we run this test?

Reply:
No, it requires a CSR to run so I ran it manually. We didn't previously run this test either, but it at least moves it into the correct place in case we want to run it automatically in the future. I think this test should/could be expanded with actual uses of these classes from Dogtag PKI, but we should leave it for a future sprint and/or QE to do.